src/views/Home/index.tsx

```diff
@@ -71,6 +78,29 @@ const HomeView = (): JSX.Element => {
     }
   }, [categories])
 
+  useEffect(() => {
+    handleGetPacks();
+  }, [])
+
+  const handleGetPacks = async () => {
+    const packs = await getPacks();
+
+    if (packs.data.success) {
+      setPackNames(packs.data.data.map<OptionType>((elem) => {
+        return { label: elem.name.substring('Polygon'.length), value: elem.identifier } as OptionType
+      }, packNames));
+
+      const previousSelection = localStorage.getItem("packSelection");
```

Comment:
usually empty line after const especially before conditional statement, also plz use simple quotes and not use ;